README.md

```diff
@@ -183,6 +183,16 @@ Any binary breaking changes will require a *MAJOR* version bump, which we will b
 cautious about. We will also consider using `organization` and package name for major 
 versioning in the future. But that decision is yet to be made. 
 
+### Adopters
+
+Here's a (non-exhaustive) list of companies that use Cats in production. Don't see yours? [Add it in a PR!](https://github.com/typelevel/cats/edit/master/README.md)
+
+- Codecentric
+- e.near
+- iHeartRadio
+- Stripe
+- Verizon
```

Comment:
Not at the time I left.  It may be true now, or in another department.

Reply:
I'll remove for now, then.

Reply:
We employed several committers and our open source libraries and Nelson are all converted, so we left a solid Cats legacy.  We just never got there ourselves. :cry: